src/pages/Login/Login.js

```diff
@@ -2,7 +2,82 @@ import React from 'react';
 import './Login.scss';
 
 const Login = () => {
-  return <div>로그인</div>;
+  return (
+    <div className="loginWrap">
```

Comment:
최상단 태그의 className은 컴포넌트와 동일하게 써주세요! 위코드 컨벤션입니다!
```suggestion
    <div className="login">
```

Reply:
완료